reference/library/src/morpheus-master/sass/modules/bullhorn-alerts/_base.scss

```diff
@@ -0,0 +1,92 @@
+.bullhorn-counter {
+    color: #fff;
+    font-weight: bold;
+    font-size: $default-font-size - 4px;
+    position: absolute;
+    padding: 0px 2px;
+    margin-top: 8px;
+    margin-left: -10px;
+    -webkit-border-radius: 2px;
+    -moz-border-radius: 2px;
+    border-radius: 2px;
+}
+
+.bullhorn-counter-red {
+    background-color: #f00;
+}
+
+.bullhorn-counter-blue {
+    background-color: #2a94c0;
+}
+
+.portal-bullhorns.qtip {
+    border: 0 none;
+    background-color: $background-color;
+    font-family: $font-family;
+    font-size: $default-font-size;
+    color: $alt-colour;
+    max-width: 450px;
+
+    .qtip-content {
+        background-color: $background-color;
+        padding: 0;
+        border: 2px solid $alt-colour;
+        color: $text-color;
+    }
+}
+
+.portal-bullhorn-alerts {
+    padding-bottom: 5px;
+    width: 450px;
+}
+    .portal-bullhorn-alert {
+        overflow: auto;
+        border: solid #E0E0E0;
+        border-width: 0px 0px 1px 0px;
+        padding: 5px;
+    }
+        .portal-bullhorn-photo {
```

Comment:
Done that on purpose to show the relationship between the styled elements. Maybe I need to rethink that.